src/pages/admin/UserList.tsx

```diff
@@ -55,6 +55,7 @@ const UserList: FC<UserListProps> = ({
       const response = await addRoleToUser(username, roleToAdd);
       if (response) {
         if (response.error) {
+          setUpdatedRoleForUsers((state) => state.filter((user) => user !== username));
```

Comment:
må fjerne loading indicator for oppdateringen som feiler, men det trengs ikke hvis det går bra siden vi da uansett henter alle brukere på nytt

Reply:
Er dette skrevet som en TODO En plass?

Reply:
nei, bruker dette for å fjerne loading state hvis en brukeroppdatering feiler. ser ingen annen måte å løse det på 🤔 
altså: kommentaren min var et forsøk på å forklare hva som blir gjort, ikke hva som gjenstår å gjøre